src/main/java/org/opensearch/performanceanalyzer/rca/store/rca/searchbackpressure/SearchBackPressureRCA.java

```diff
@@ -0,0 +1,521 @@
+                new MinMaxSlidingWindow(SLIDING_WINDOW_SIZE_IN_MINS, TimeUnit.MINUTES, false);
+
+        // sliding window for JVM
+        this.shardJVMCancellationSlidingWindow =
+                new SlidingWindow<>(SLIDING_WINDOW_SIZE_IN_MINS, TimeUnit.MINUTES);
+        this.taskJVMCancellationSlidingWindow =
+                new SlidingWindow<>(SLIDING_WINDOW_SIZE_IN_MINS, TimeUnit.MINUTES);
+
+        LOG.info("SearchBackPressureRCA initialized");
+    }
+
+    /*
+     * operate() is used for local build
+     * This will compute the flow units from other hosts in the cluster
+     * for a given Metric and try to send the subscription requests
+     * to stale or new hosts in cluster if need be
+     */
+    @Override
+    public void generateFlowUnitListFromWire(FlowUnitOperationArgWrapper args) {
+        final List<FlowUnitMessage> flowUnitMessages =
+                args.getWireHopper().readFromWire(args.getNode());
+        final List<ResourceFlowUnit<HotNodeSummary>> flowUnitList = new ArrayList<>();
+        LOG.info("rca: Executing fromWire: {}", this.getClass().getSimpleName());
+        for (FlowUnitMessage flowUnitMessage : flowUnitMessages) {
+            flowUnitList.add(ResourceFlowUnit.buildFlowUnitFromWrapper(flowUnitMessage));
+        }
+        setFlowUnits(flowUnitList);
+    }
+
+    /*
+     * operate() evaluates the current stats against threshold
+     * Unhealthy Flow Units is a marker that this resource at current instance is not healthy
+     * Autotune decision would be made by downstream classes
+     */
+    @Override
+    public ResourceFlowUnit<HotNodeSummary> operate() {
+        currentIterationNumber += 1;
+        ResourceContext context = null;
+        long currentTimeMillis = System.currentTimeMillis();
+
+        // read key metrics into searchBackPressureRCAMetric for easier management
+        SearchBackPressureRCAMetric searchBackPressureRCAMetric = getSearchBackPressureRCAMetric();
+
+        // print out oldGenUsed and maxOldGen
+        LOG.info(
+                "SearchBackPressureRCA: oldGenUsed: {} maxOldGen: {}, heapUsedPercentage: {}, searchbpShardCancellationCount: {}, searchbpTaskCancellationCount: {}, searchbpJVMShardCancellationCount: {}, searchbpJVMTaskCancellationCount: {}",
+                searchBackPressureRCAMetric.getUsedHeap(),
+                searchBackPressureRCAMetric.getMaxHeap(),
+                searchBackPressureRCAMetric.getHeapUsagePercent(),
+                searchBackPressureRCAMetric.getSearchbpShardCancellationCount(),
+                searchBackPressureRCAMetric.getSearchbpTaskCancellationCount(),
+                searchBackPressureRCAMetric.getSearchbpJVMShardCancellationCount(),
+                searchBackPressureRCAMetric.getSearchbpJVMTaskCancellationCount());
+
+        updateAllSlidingWindows(searchBackPressureRCAMetric, currentTimeMillis);
+
+        LOG.info("SearchBackPressureRCA currentIterationNumber is {}", currentIterationNumber);
```

Comment:
we can change some of the `LOG.info` messages to `LOG.debug`. `LOG.info` should only contains essential information to store in the service log.

Reply:
Change unnecessary log message level to debug